code/game/antagonist/station/loyalist.dm

```diff
@@ -21,13 +21,15 @@ var/datum/antagonist/loyalists/loyalists
 	initial_spawn_target = 4
 
 	// Inround loyalists.
-	faction_role_text = "Loyalist"
-	faction_descriptor = "Company"
+	faction_role_text = "Fellow"
+	faction_descriptor = "Fellowship"
 	faction_verb = /mob/living/proc/convert_to_loyalist
-	faction_welcome = "Preserve NanoTrasen's interests against the traitorous recidivists amongst the crew. Protect the heads of staff with your life."
-	faction_indicator = "loyal"
-	faction_invisible = 1
+	faction_welcome = "You have joined a budding fellowship under the forward-thinking lead of a Fellowship leader. Follow their instructions and try to achieve the Fellowships goals."
+	faction_indicator = "fellowship"
+	faction_invisible = FALSE
+
 	restricted_jobs = list("AI", "Cyborg")
+	protected_jobs = list("Security Officer", "Security Cadet", "Warden", "Detective", "Forensic Technician", "Head of Personnel", "Chief Engineer", "Research Director", "Chief Medical Officer", "Captain", "Head of Security", "Internal Affairs Agent")
```

Comment:
why's IAA in here?

Reply:
Legacy code that I didn't adjust I think, this code hasn't been touched since the IAA was replaced I think. I'll remove it from the list.

Reply:
Should be fixed X2.